aplib.net-demo/Assets/Scripts/Entities/Weapons/MeleeWeapon.cs

```diff
@@ -39,6 +34,20 @@ private void Awake()
         EnemiesWithinRange();
     }
 
+    /// <summary>
+    /// Initialize the weapon with the damage, height, radius and target tag.
+    /// </summary>
+    /// <param name="damage">The amount of damage the weapon will deal.</param>
+    /// <param name="height">The height of the hitzone in world units.</param>
+    /// <param name="radius">The radius of the hitzone in world units.</param>
+    /// <param name="targetTag">The tag of the target.</param>
+    public void Initialize(int damage, string targetTag, float height, float radius)
+    {
+        base.Initialize(damage, targetTag);
```

Comment:
Not sure but it says base is not necessary here, also it is not an override

Reply:
I put base here to make clearer that it is the method from the abstract weapon class, but I can remove that if not necessary. It is not an override because it's a different method with a different access modifier.

Reply:
I agree with @JoenvandeVorle. Clarity is love, clarity is life.